packages/smart-contracts/test/contracts/BatchNoConversionErc20Payments.test.ts

```diff
@@ -100,82 +112,105 @@ describe('contract: batchNoConversionPayments: ERC20', () => {
 
   describe('Batch working well: right args, and approvals', () => {
     it('Should pay 3 ERC20 payments with paymentRef and pay batch fee', async () => {
-      await token1.connect(owner).transfer(spender3Address, 1000);
-      await token1.connect(spender3).approve(batchAddress, 1000);
+      await token1.connect(owner).transfer(spender3Address, 10000);
```

Comment:
Are you testing skipFeeUSDLimit somewhere? If yes the test is hard to find, it would be nice to add `skipFeeUSDLimit` in the test case name.